libmamba/src/core/fetch.cpp

```diff
@@ -159,12 +95,12 @@ namespace mamba
     {
         auto& ctx = Context::instance();
 
-        if (!ctx.curl_initialized)
+        if (!m_curl_initialized)
```

Comment:
I think there is a risk that we do many times the same thing here since `ctx.curl_initialized` is a global, shared variable, while `m_curl_initialized` is a data member specific to a DownloadTarget data member.

This might not be a big deal here since we just set another variable in the context, but it might become later if we add stuff in this function.

What is the motivation for removing this variable from the context? Would it make sense to have it as part of a "DownloaderOption" subobject in the Context? (that could be done in another PR, but the current variable would remain in the context in this PR).

Reply:
Well I actually found out that it's not used anywhere but here in the `fetch`... so no reason to keep it in the context! Do you see any, or think it would be used somewhere else?

Reply:
I think it should live with the ssl_verify variable and maybe move to CURLSetup on the long run. This should be set once and for good. Keeping it in the context is ok for now.